app/web/features/connections/friends/useFriendList.test.tsx

```diff
@@ -63,53 +67,16 @@ describe("when the listFriends query succeeds", () => {
     await waitForNextUpdate();
 
     expect(result.current).toMatchObject({
-      data: [
-        {
-          avatarUrl: "",
-          name: "Funny Dog",
-          userId: 2,
-          username: "funnydog",
-        },
-        undefined,
-      ],
+      data: [],
```

Comment:
What's happening here? Just seems weird it went from data to empty array data when no major logic changed.

Reply:
This used to test what happens if you request two users and one is loading, the other has loaded. We don't do that anymore because we load all the users in one chunk. So I changed this test to test what happens if all the users are still loading, I don't think that's tested elsewhere.

...now I'm looking at it again and wondering if I did something funny since there's another loading test above.

Reply:
Ah okay gotcha, I think one is testing when the getLiteUsers query is loading and the other is testing when useFriendList itself is loading to make sure we handle both scenarios.